src/Features/Core/Portable/MakeMethodAsynchronous/AbstractMakeMethodAsynchronousCodeFixProvider.KnownTypes.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.MakeMethodAsynchronous
+{
+    internal abstract partial class AbstractMakeMethodAsynchronousCodeFixProvider
+    {
+        internal readonly struct KnownTypes
+        {
+            public readonly INamedTypeSymbol _taskType;
```

Comment:
Nit: Consider using auto-props instead.

Reply:
in general, IDE is totally fine with readonly-fields for internal simple-data structs.  The props don't really buy anything since we have no need to change the impl over time.  or, in other words, if we did need to change the impl, we own all the consumers, so we can just change them.  so, the readonly field is just the simplest way of saying "this is just raw data, and nothing more".

Reply:
I agree. I think I prefer simple field for now.
Thanks